.github/workflows/ci.yml

```diff
@@ -35,7 +35,8 @@ jobs:
       - uses: dtolnay/rust-toolchain@master
         with:
           toolchain: ${{ matrix.rust }}
-      - run: cargo check ${{ matrix.features }}
+          components: ${{ matrix.rust == 'stable' && 'clippy' || '' }}
```

Comment:
Let's only run clippy on the msrv toolchain. Clippy recommendations that violate msrv are not actionable.

Reply:
@jswrenn again, we recommend option B. Call it and we'll do it.